website_snippet_country_dropdown/test/__init__.py

```diff
@@ -0,0 +1,2 @@
+
+import test_snippet_country_dropdown
```

Comment:
You need to put `from . import` before.

You also need the license boilerplate [as you can see in the module template](https://github.com/OCA/maintainer-tools/blob/master/template/module/tests/test_something.py#L1-L3).